### PR TITLE
Add pause tracking keybind & protobuf command

### PR DESCRIPTION
**Pull request by ButterscotchV**
Toggle paused tracking with a keybind and/or feeder app, very nice. Also now logs when it's set, which includes the current state. Requested feature from a friend :3

SolarXR and GUI are updated to request and listen for the state of tracking pause from the server, so it should update when using keybinds and stuff!

Depends on https://github.com/SlimeVR/SolarXR-Protocol/pull/126

### CODE REVIEW

**Review by ImUrX — APPROVED** [review]

**Review by ImUrX — CHANGES_REQUESTED** [review]
oops

File: server/core/src/main/java/dev/slimevr/VRServer.kt [hunk 1]
```diff
@@ -324,6 +332,24 @@ class VRServer @JvmOverloads constructor(
...
+
+	fun scheduleTogglePauseTracking(sourceName: String?, delay: Long) {
+		val pauseTrackingTask: TimerTask = object : TimerTask() {
+			override fun run() {
+				queueTask { humanPoseManager.togglePauseTracking(sourceName) }
+			}
+		}
+		timer.schedule(pauseTrackingTask, delay)
+	}
+
```
[the remark this answers is not in the document]
Comment (ImUrX) [reply server/core/src/main/java/dev/slimevr/VRServer.kt:352]: thats preferable tbh, it looks pretty bad this way XD

[the remark this answers is not in the document]
Comment (ButterscotchV) [reply server/core/src/main/java/dev/slimevr/VRServer.kt]: Should be good now https://github.com/SlimeVR/SlimeVR-Server/pull/867/commits/e66664081ab39ce56cbf86930c8330e124b28e7f

**Review by ImUrX — APPROVED** [review]

File: server/core/src/main/java/dev/slimevr/config/KeybindingsConfig.java [hunk 2]
```diff
@@ -8,12 +8,16 @@ public class KeybindingsConfig {
 
 	private String mountingResetBinding = "CTRL+ALT+SHIFT+I";
 
+	private String pauseTrackingBinding = "CTRL+ALT+SHIFT+O";
```
Comment (Louka3000) [inline server/core/src/main/java/dev/slimevr/config/KeybindingsConfig.java:11]: this should totally be P for Pause instead of O

**Review by Louka3000 — APPROVED** [review]

File: server/core/src/main/java/dev/slimevr/config/KeybindingsConfig.java (hunk 2 above)
Comment (ButterscotchV) [reply server/core/src/main/java/dev/slimevr/config/KeybindingsConfig.java:11]: I just followed the row of keys :P

**Review by Eirenliel — APPROVED** [review]

**Pull request merged**